packages/backend/src/workers/provider/LlamaCppPython.ts

```diff
@@ -134,6 +134,24 @@ export class LlamaCppPython extends InferenceProvider {
             PathInContainer: '/dev/dri',
             CgroupPermissions: '',
           });
+          break;
+        case VMType.UNKNOWN:
+          // Only supports NVIDIA
+          if (gpu.vendor !== GPUVendor.NVIDIA) break;
+
+          supported = true;
+          devices.push({
+            PathOnHost: 'nvidia.com/gpu=all',
```

Comment:
This need the `NVIDIA Container Device Interface` installed ?

Reply:
@axel7083 do you mean detection on the local machine instead of podman machine? It looks like to me that on Linux there is no podman machine as podman runs natively on the local machine.

Reply:
We might be able to do that by looking for the files which are generated. From

https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/cdi-support.html#procedure

on my Fedora system that is `/etc/cdi/nvidia.yaml`

Since it says it deps on the container engine you use, we could possibly just check in `/etc/cdi/nvidia.yaml` on the assumption that is what is for podman. We might later add additional places to check if needed later on.

If that check would be enough I can look at adding it.